index.html

```diff
@@ -1917,8 +1917,11 @@ <h4>Representing Time</h4>
 misinterpretation of time values, by utilizing the
 <code>dateTimeStamp</code> construction first established by the [<a
 data-cite="XMLSCHEMA11-2#dateTimeStamp">XMLSCHEMA11-2</a>] specification. In
-order to reduce misinterpretations between different regions, all times MUST be
-specified as offsets against Universal Coordinated Time (UTC).
+order to reduce misinterpretations between different time zones, any time
+value MUST specify a time zone offset relative to Universal Coordinated Time
+(UTC). Examples of valid time zone offsets relative to UTC include
+`Z`, `+01:00`, `-08:00`, and `+14:00`. See the regular expression at the
+end of this section for a formal definition of all acceptable values.
```

Comment:
This is fine, albeit a lot of detail for your spec. 

This doesn't say what happens when a given VC credential ignores it (is the credential invalid??) One work around for this would be to specify that time values must include an offset (with UTC preferred) and separately specify that values without an offset will be interpreted as UTC. 

Also, note that temporal serializations are coming, which include true time zone IDs (offsets have many problems). The I-D isn't an RFC yet, so you can't reference it here, but be aware of it for revisions in the future. The WG there is called [SEDATE](https://datatracker.ietf.org/doc/draft-ietf-sedate-datetime-extended/)

Reply:
> This is fine, albeit a lot of detail for your spec.

Yes. :P

When people raise issues, we try to be crystal clear in the changes, and that has led to a verbose specification (that some people are cranky about)... but if we remove the detail, we end up not addressing the concerned party's needs/concerns. Such is the specification editing process. :)

> This doesn't say what happens when a given VC credential ignores it (is the credential invalid??) One work around for this would be to specify that time values must include an offset (with UTC preferred) and separately specify that values without an offset will be interpreted as UTC.

Good idea, that covers all of our bases AFAICT. I'll implement it in an upcoming modification to this PR.

It would allow us to weaken the existing MUST to a SHOULD, but still ensure a single "correct" way to interpret all time values. I'm on the fence wrt. weakening the existing MUST to a SHOULD -- thoughts, @jyasskin and @aphillips? I could go either way.

> Also, note that temporal serializations are coming, which include true time zone IDs (offsets have many problems). The I-D isn't an RFC yet, so you can't reference it here, but be aware of it for revisions in the future. The WG there is called [SEDATE](https://datatracker.ietf.org/doc/draft-ietf-sedate-datetime-extended/)

Interesting... if we were to switch to IXDTF syntax in the future, we'd have to move away from XML Schema dateTimeStamp (and the ref to that spec) and move back to RFC3339 (which we had to move away from for reasons I can't remember right now). That is definitely a VC v3.0 thing, so let's let things settle in SEDATE-land before we make any changes. I don't forsee any spec issues if we had to switch given that dateTimeStamps are a subset of IXDTF syntax. The biggest issue is deployed infrastructure choking on IXDTF datetimes... we'd have to make sure IXDTF parsing is in a large variety of datetime libraries before shipping that feature.

Reply:
> It would allow us to weaken the existing MUST to a SHOULD, but still ensure a single "correct" way to interpret all time values. I'm on the fence wrt. weakening the existing MUST to a SHOULD -- thoughts, @jyasskin and @aphillips? I could go either way.

I'd be reluctant to reduce MUST to SHOULD as well. Perhaps something like:

```suggestion
All time values MUST be specified as dateTimeStamp, either in UTC (denoted by `Z`) 
or with a time zone offset relative to Universal Coordinated Time. 
Time values incorrectly serialized without an offset MUST be interpreted as UTC.
```

(above suggestion is not meant to be merged, as it needs references and such)

> Interesting... if we were to switch to IXDTF syntax in the future, we'd have to move away from XML Schema dateTimeStamp (and the ref to that spec) and move back to RFC3339 (which we had to move away from for reasons I can't remember right now). That is definitely a VC v3.0 thing, so let's let things settle in SEDATE-land before we make any changes. I don't forsee any spec issues if we had to switch given that dateTimeStamps are a subset of IXDTF syntax. The biggest issue is deployed infrastructure choking on IXDTF datetimes... we'd have to make sure IXDTF parsing is in a large variety of datetime libraries before shipping that feature.

I suspect Temporal will be as popular in JS as it has been in other programming environments. This will likely lead to various serializations getting updated too. RFC3339 is just ahead of the curve here. You have the right idea: wait for the dust to settle.

Reply:
Applied your changes in 91339e127d533724a276663a805ad7db882043d7.